pkg/dataplane/util/ansibleee.go

```diff
@@ -152,32 +151,6 @@ func (a *EEJob) JobForOpenStackAnsibleEE(h *helper.Helper) (*batchv1.Job, error)
 	if len(a.ServiceAccountName) > 0 {
 		job.Spec.Template.Spec.ServiceAccountName = a.ServiceAccountName
 	}
-	// Set primary inventory if specified as string
-	existingInventoryMounts := ""
-	if len(a.Inventory) > 0 {
```

Comment:
nothing sets a.Inventory afaict, which is why we can drop this. It must be a leftover from when we migrated from openstack-ansibleee-operator. If that's correct, then lgtm.

Reply:
Yeah, I don't think there is anything setting it. I should probably drop the param from the struct as well. Updated the PR to remove it.